src/main/java/com/kodilla/ecommercee/controller/OrderController.java

```diff
@@ -1,4 +1,41 @@
 package com.kodilla.ecommercee.controller;
 
+import com.kodilla.ecommercee.dto.OrderDto;
+import org.springframework.web.bind.annotation.*;
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.http.MediaType;
+
+@RestController
+@RequestMapping("/v1/order")
 public class OrderController {
```

Comment:
Akceptuję to, bo zakładam, że od teraz, po wczorajszej rozmowie, w pull requestach będzie już tylko to, co powinno w nich być.